pkg/printers/internalversion/printers.go

```diff
@@ -588,6 +589,13 @@ func AddHandlers(h printers.PrintHandler) {
 	}
 	h.TableHandler(storageVersionColumnDefinitions, printStorageVersion)
 	h.TableHandler(storageVersionColumnDefinitions, printStorageVersionList)
+
+	scaleColumnDefinitions := []metav1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "DesiredReplicas", Type: "integer", Description: autoscalingv1.ScaleSpec{}.SwaggerDoc()["replicas"]},
+		{Name: "AvailableReplicas", Type: "integer", Description: autoscalingv1.ScaleStatus{}.SwaggerDoc()["replicas"]},
```

Comment:
sorry, should have commented on this earlier -- wdyt about naming this to just `Available` and `Desired` since replicas is implicit? It would also follow the same pattern as Deployment then

Reply:
I am good with this change. Will change it.